index.html

```diff
@@ -46,6 +46,42 @@ <h1>I’m Kaweesi Glad to see you!</h1>
           </li>
         </ul>
       </section>
+      <div class="wrapper2">
+        <section class="tonic">
+          <div class="imgez"><img src="./images/snap.png" alt="snap" /></div>
+          <div>
+            <h2>Tonic</h2>
+          </div>
```

Comment:
@Kaweesi-Matia I'll suggest that you do not edit others' comments on your repository in as much as they're not abusive/offensive. This will portray that you respect their opinions. Kindly reply the comments instead.